modules/retail/pages/retail-install-setup.adoc

```diff
@@ -131,88 +150,38 @@ For this to work [systemitem]``DHCP`` service for the network terminal is connec
-- resolve [systemitem]``salt`` CNAME, if successful then resolved value is used as Salt master
-- use [systemitem]``salt`` as a Salt master
-
-Once Salt master is determined, Salt client configuration is generated and started.
-
-[IMPORTANT]
-====
-Using fully qualified domain name in [systemitem]``MASTER`` or [systemitem]``salt`` CNAME is important.
-
-If used fully qualified domain name is different from fully qualified domain name of branch server proxy known to {productname}, {saltboot} may work correctly, however proxy detection of terminal will not work.
-====
-
-==== Download Server Discovery
-
-Before POS image is downloaded to the terminal, download server discovery is done to find where to download image from:
-
-- [systemitem]``saltboot_download_server`` pillar is set for terminal, then its value is used
-- [systemitem]``saltboot:download_server`` pillar is set for terminal, then its value is used
-- resolve [systemitem]``ftp`` hostname
-
-Value obtained by download server discovery is then used together with POS image pillar to fetch correct image from correct location.
-
-
-=== Terminal Partitioning and Image Selection
-
-Last piece for {saltboot} is to provide partitioning for terminal.
+{saltboot} requires instructions how to partitioning terminal harddisk and what image to deploy.
 This is done individually for each hardware type of terminals.
-For more information about hardware types, see xref:retail:retail-deploy-terminals.adoc[].
+For more information about hardware types and partitioning, see xref:retail:retail-deploy-terminals.adoc[].
 
 Above mentioned steps are mandatory minimum for successful {saltboot} deployment.
 For configuration examples, see xref:retail:example-configurations.adoc[].
 
 
 == Synchronize Images to the Branch Server
 
-The OS image you use on the {productname} server must be synchronized for use to the branch server.
-You can do this with the Salt [command]``image-sync`` state, part of the [systemitem]``Image Synchronization Formula``.
-
-.Procedure: Synchronizing Images to the Branch Server
-
-. On the {productname} server, run this command:
-+
-----
-salt <branch_server_minion_id> state.apply image-sync
-----
-. The image details will be transferred to [path]``/srv/saltboot`` on the branch server.
+{productname} {smr} {productnumber} no longer need manual image synchronization, all images are available to all clients automatically.
 
-You can also set synchronization to run automatically on the branch server.
-Configure the image synchronization formula to apply the highstate regularly.
-For more information about [systemitem]``Image Synchronization Formula``, see xref:specialized-guides:salt/salt-formula-imagesync.adoc[].
+This may not be always desired, for example to allow images gradual deployments across all branches. For a way how to limit image deployment, see xref:retail:retail-best-practices.adoc[].
```

Comment:
```suggestion
This may not be always desired, for example to allow images gradual deployments across all branches.
For a way how to limit image deployment, see xref:retail:retail-best-practices.adoc[].
```